Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6.5
+
+COPY ./newsAnalysis /app/newsAnalysis
+COPY ./requirements.txt /app/requirements.txt
+
+WORKDIR /app
+RUN pip install -r requirements.txt
```

Comment:
is it possible to add matplotlib==2.0.2 in the requirements file to avoid an error from the ImagePlotter?
and for the nltk module besides installing it is also required to also run a python session with:
import nltk
nltk.download('punkt')
Would be great to add that here